plugins/chat/assets/javascripts/discourse/lib/get-user-chat-separate-sidebar-mode.js

```diff
@@ -0,0 +1,9 @@
+export function getUserChatSeparateSidebarMode(user) {
+  const mode = user.get("user_option.chat_separate_sidebar_mode");
+
+  return {
+    never: "never" === mode,
+    always: "always" === mode,
+    fullscreen: "fullscreen" === mode,
```

Comment:
Love ❤️ the string on the left (instead of the variable) and the strict equality checks 👌

Reply:
Something interesting to note on this that david told me, is that even if it's in a lib, it will work with ember tracking.

```
get something() {
  return getUserSeparatedSidebarMode(this.currentUser);
}
```

Would cause a re-render if `this.currentUser.user_option.xxxx`  changes